composer.json

```
@@ -81,7 +81,10 @@
     "lint:webp-uploads":           "@lint -- ./plugins/webp-uploads --standard=./plugins/webp-uploads/phpcs.xml.dist",
     "phpstan": "phpstan analyse --memory-limit=2048M",
     "test": "phpunit",
-    "test-multisite": "@test -c tests/multisite.xml",
+    "test-multisite": [
+      "php bin/generate-multisite-phpunit.php",
+      "@test -c phpunit-multisite.xml"
+    ],
```

```suggestion
    "test-multisite": [
      "WP_MULTISITE=1 phpunit --exclude-group=ms-excluded"
    ],
```

It should be sufficient, and we don't need a separate config for multisite until we have a strong use case IMO.

---

It's working fine with these changes. Waiting for @westonruter feedback.

---

But what about my PHP code?!? 

😿

---

😢